packages/base64/package.json

```diff
@@ -31,8 +31,8 @@
     "build": "exit 0",
     "cover": "c8 ava",
     "lint": "yarn lint:types && yarn lint:js",
-    "lint-fix": "eslint --fix '**/*.js'",
-    "lint:js": "eslint '**/*.js'",
+    "lint-fix": "eslint --fix .",
```

Comment:
Note to self: according to [eslint doc](https://eslint.org/docs/user-guide/command-line-interface#--ext), `.js` is the default extension so this is purely equivalent.